tools/oav-traffic-converter/.gitignore

```diff
@@ -0,0 +1,2 @@
+input-example/
```

Comment:
Do we need a custom gitignore in the tools directory?

Reply:
I'm of the opinion that each project should put it's special case files in a local gitignore, but not technically. I'll just adjust.